crates/invoker-impl/src/lib.rs

```diff
@@ -1090,6 +1090,11 @@ where
             .remove_invocation_with_epoch(partition, &invocation_id, invocation_epoch)
         {
             debug_assert_eq!(invocation_epoch, ism.invocation_epoch);
+
+            if self.is_service_down_error(&error) {
+                counter!(INVOKER_SERVICE_UNREACHABLE_ERRORS, "service" => ism.invocation_target.service_name().to_string()).increment(1);
```

Comment:
I think we need to have both the deployment id and the service name. The risk is that it this will be a high cardinality metric.